var/spack/repos/builtin/packages/kadath/package.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+from spack import *
+
+
+class Kadath(CMakePackage):
+    """KADATH SPECTRAL SOLVER
```

Comment:
```suggestion
    """KADATH SPECTRAL SOLVER.
```
`spack info` removes newlines, so this would look like a single run-on sentence.